zanataclient/zanatalib/projectutils.py

```diff
@@ -142,3 +144,82 @@ def dict2xml(tag, dict):
             child.text = str(val)
             elem.append(child)
         return ToolBox.prettify(elem)
+
+
+class FileMappingRule(object):
+    """
+    Build translation's paths for pull and push considering rules
+
+    The mapping rules configuration is optional in zanata.xml.
+    If not specified, standard rules are applied according to project type.
+    """
+
+    project_filemapping_default_config = {
+        'gettext': '{path}/{locale_with_underscore}.{extension}',
+        'podir': '{locale}/{path}/{filename}.{extension}',
```

Comment:
Should this default path not be 
    {path}/{locale}/{filename}.{extension}

for it to replicate something like 
    - po
        - en
              - file1.po
              - file2.po
        -no
              - file1.po
              - file2.po

And if so, I would assume the same woul be for the "file" bellow